core/blockstore/store.ts

```
@@ -29,7 +29,7 @@ import {
   UnsubscribeResult,
   CommitQueueIf,
 } from "@fireproof/core-types-blockstore";
-import { Falsy, isNotFoundError, PARAM, StoreType, SuperThis } from "@fireproof/core-types-base";
+import { Falsy, isNotFoundError, PARAM, StoreType, SuperThis, throwFalsy } from "@fireproof/core-types-base";
```

ESLint

[error] 'throwFalsy' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)